src/blockdata/script.rs

```diff
@@ -371,7 +380,11 @@ impl Script {
 
     /// Generates P2TR for script spending path using an internal public key and some optional
     /// script tree merkle root.
-    pub fn new_v1_p2tr<C: Verification>(secp: &Secp256k1<C>, internal_key: UntweakedPublicKey, merkle_root: Option<TapBranchHash>) -> Script {
+    pub fn new_v1_p2tr<C: Verification>(
+        secp: &Secp256k1<C>,
+        internal_key: UntweakedPublicKey,
+        merkle_root: Option<TapBranchHash>,
```

Comment:
Lolz, this can be configured i think. If i remember correctly we can set max length for functions. FTR writing C in the kernel as super annoying for anyone who is OCD about uniformity when it comes to function args because each subsystem (and sometimes each file) uses its own format. While I kind of agree with you, its really hard to find a uniform format for parameters that is _always_ right.

Reply:
I really dislike the "compressed" version and prefer the "everything one its own line". Perhaps Matt does not read things in tabular form where you can clearly see the number of arguments and distinguish names from types, but I do.